weblab/templates/datasets/dataset_list.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+{% load datasets %}
+
+{% block title %}Your datasets - {% endblock title %}
+
+{% block content %}
+
+
+    {% can_create_dataset as permission %}
+    {% if permission %}
+    <small><a href="{{ type|url_new }}" id="add{{ type }}link" class="pointer">Create a new dataset</a></small>
```

Comment:
Probably don't need the `type` in the file as its always used for experimental datasets.